src/NHSDPublicBrowseAcceptanceTestsSpecflow/Steps/BrowseSolutions/ViewASolution.cs

```diff
@@ -105,10 +105,9 @@ public void ThenContactDetails()
         [Then(@"Last Updated Date")]
         public void ThenLastUpdatedDate()
         {
-            var lastUpdated = SolutionDetails.LastUpdated.ToString("yyyy-MM-dd");
-            var actualLastUpdated = _test.pages.ViewASolution.GetSolutionLastUpdated();
-            actualLastUpdated = Convert.ToDateTime(actualLastUpdated).ToString("yyyy-MM-dd");
-            actualLastUpdated.Should().Be(lastUpdated);
+            var lastUpdated = SolutionDetails.LastUpdated.ToString("dd MMMM yyyy");
```

Comment:
might as well use the ConvertDateToLongDateTime method, keep the date format in one place

Reply:
That adds the time onto the end of the date, which is not correct for the scenario under test

Reply:
I don't understand what you mean.
`var lastUpdated = ConvertDateToLongDateTime(SolutionDetails.LastUpdated.ToString());` seems like it will work

Reply:
It's converting to a string to convert back to a date to convert into a string. It's needless when it's only done in 1 place. If we do start needing to verify this in multiple places, I will refactor

Reply:
Can you create a class variable called longDateFormat = "dd MMMM yyyy" and let's use that instead of the literal string. And then count how many times that ConvertDateToLongDateTime is being used, if it's only being used you might as well delete it then.